assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -30,6 +30,7 @@ import { __ } from '@wordpress/i18n';
 import useMedia from '../../app/media/useMedia';
 import { useConfig } from '../../app/config';
 import { useSnackbar } from '../../app/snackbar';
+import { useAPI } from '../../app/api';
```

Comment:
Seems like this import causes some troubles with the tests, because the API provider imports from `../app`. Reminds me of #1290.

```
    TypeError: Cannot read property 'default' of undefined



      at Object.TextInput (assets/src/edit-story/components/form/index.js:70:18)
      at Object.<anonymous> (assets/src/edit-story/components/panels/shared/expandedTextInput.js:26:31)
      at Object.<anonymous> (assets/src/edit-story/components/panels/shared/index.js:16:1)
      at Object.<anonymous> (assets/src/edit-story/components/panels/imageAccessibility.js:31:1)
      at Object.<anonymous> (assets/src/edit-story/components/panels/index.js:23:1)
      at Object.<anonymous> (assets/src/edit-story/elements/text/index.js:22:1)
      at Object.<anonymous> (assets/src/edit-story/elements/index.js:31:1)
      at Object.<anonymous> (assets/src/edit-story/components/canvas/useInsertElement.js:26:1)
      at Object.<anonymous> (assets/src/edit-story/components/dropTargets/provider.js:28:1)
      at Object.<anonymous> (assets/src/edit-story/components/dropTargets/index.js:17:1)
      at Object.<anonymous> (assets/src/edit-story/app/index.js:35:1)
      at Object.<anonymous> (assets/src/edit-story/app/api/apiProvider.js:33:1)
      at Object.<anonymous> (assets/src/edit-story/app/api/index.js:17:1)
      at Object.<anonymous> (assets/src/edit-story/components/mediaPicker/useMediaPicker.js:33:1)
      at Object.<anonymous> (assets/src/edit-story/components/mediaPicker/index.js:17:1)
      at Object.<anonymous> (assets/src/edit-story/components/form/media.js:33:1)
      at Object.<anonymous> (assets/src/edit-story/components/form/index.js:22:1)
      at Object.<anonymous> (assets/src/edit-story/components/richText/formatters/test/weight.js:20:1)
```

Reply:
Yay, a5bdbcfd33792f1defa09b829e735f9c784f659c did the trick!